protocol/x/subaccounts/keeper/transfer_test.go

```diff
@@ -1269,14 +1300,57 @@ func TestTransferFeesToFeeCollectorModule(t *testing.T) {
 			collateralPoolAddr:                  types.ModuleAddress,
 			expectedSubaccountsModuleAccBalance: big.NewInt(1000),
 			expectedFeeModuleAccBalance:         big.NewInt(1000),
+			marketMapperAccBalance:              big.NewInt(0),
+			marketMapperAddr:                    constants.AliceAccAddress,
+			expectedMarketMapperAccBalance:      big.NewInt(0),
+		},
+		"success - distribute fees to market mapper and fee collector": {
+			asset:                               *constants.Usdc,
+			feeModuleAccBalance:                 big.NewInt(2500),
+			subaccountModuleAccBalance:          big.NewInt(600),
+			marketMapperAccBalance:              big.NewInt(0),
+			quantums:                            big.NewInt(500),
+			expectedSubaccountsModuleAccBalance: big.NewInt(100),  // 600 - 500
+			expectedFeeModuleAccBalance:         big.NewInt(2950), // 500 + 2500 - 50
```

Comment:
nit: more clear that 2500 is the original balance if you reorder to `2500 + 500 - 50`